### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'decode'

### ISSUE THREAD

**Issue by orsenthil** [DTVD/rainbowstream#121]
```
$ rainbowstream -iot
/Users/skumaran/rainbow/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
You are running latest version (1.2.7)

We have connection problem with twitter REST API right now :(
Traceback (most recent call last):
  File "/Users/skumaran/rainbow/bin/rainbowstream", line 8, in <module>
    load_entry_point('rainbowstream==1.2.7', 'console_scripts', 'rainbowstream')()
  File "/Users/skumaran/rainbow/lib/python2.7/site-packages/rainbowstream/rainbow.py", line 2198, in fly
    detail_twitter_error(e)
  File "/Users/skumaran/rainbow/lib/python2.7/site-packages/rainbowstream/util.py", line 20, in detail_twitter_error
    info = "Error: " + twitterException.response_data.decode('utf8')
AttributeError: 'dict' object has no attribute 'decode'
```

**Comment by kramimus** [comment]
Also just saw this in 1.3.0 when trying to tweet:

Traceback (most recent call last):
  File "/home/mark/.virtualenvs/chat/bin/rainbowstream", line 9, in <module>
    load_entry_point('rainbowstream==1.3.0', 'console_scripts', 'rainbowstream')()
  File "/home/mark/.virtualenvs/chat/lib/python2.7/site-packages/rainbowstream/rainbow.py", line 2240, in fly
    listen()
  File "/home/mark/.virtualenvs/chat/lib/python2.7/site-packages/rainbowstream/rainbow.py", line 1965, in listen
    detail_twitter_error(e)
  File "/home/mark/.virtualenvs/chat/lib/python2.7/site-packages/rainbowstream/util.py", line 20, in detail_twitter_error
    info = "Error: " + twitterException.response_data.decode('utf8')
AttributeError: 'dict' object has no attribute 'decode'

**Comment by DTVD** [comment]
Hi @kramimus @orsenthil Can you guys specify which Python version are you using ? Thanks !

**Comment by orsenthil** [comment]
This is 2.7.10. I figured out the problem. My OAuth was unsuccessful and it resulted in this cryptic error. I will submit a pull request for this shortly.

**Comment by DTVD** [comment]
I'm also trying to get rid of this errors display. It seems that the behavior of `Python Twitter Tool` changed recently and does not compatible with my code at this time. Will update shortly too.

**Comment by DTVD** [comment]
This is fixed in 5b5f261a7b3552b9bca72bdc72317af1566d243e and the newest version (1.3.1)

**Comment by orsenthil** [comment]
Thanks! That should work. I managed to create a pull request too which keeps your previous conditions intact. https://github.com/DTVD/rainbowstream/pull/125  (This is based on top of 5b5f261 and you can consider pulling it).
